controllers/ControllerPost.php

```diff
@@ -24,18 +25,24 @@ public function renderPost(){
 		$comments = new CommentsManager();
 		$thePost = $post->getPost($this->slug);
 		$allComments = $comments->postComment($this->slug);
-		if (!empty($_SESSION)){
+		$keyPost = 'post';
+
+		(new Session)->setKey($keyPost, ['id' => $this->slug]);
+		$key = (new Session)->getKey('user');
+
+		if (!empty($key)){
+			$modifComment = $key['id'];
 			$logged = true;
 			echo $this->twig->render( 'post.twig', ['thePost' => $thePost,
 			                                        'allComments' => $allComments,
 			                                        'logged'=> $logged,
-			                                        'user'=> $_SESSION['user']['pseudo']]);
+			                                        'user'=> $key['pseudo'],
+													 'modifComment' => $modifComment]);
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$modifComment'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)